src/main/drivers/pwm_output_stm32f3xx.c

```diff
@@ -196,8 +188,6 @@ void pwmDigitalMotorHardwareConfig(const timerHardware_t *timerHardware, uint8_t
     DMA_InitStructure.DMA_M2M = DMA_M2M_Disable;
 
     DMA_Init(channel, &DMA_InitStructure);
-
-    DMA_ITConfig(channel, DMA_IT_TC, ENABLE);
```

Comment:
This is what I added
https://github.com/betaflight/betaflight/commits/blckmn-remove_it_dshot